src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -61,6 +61,7 @@ function getSentryIntegrations(hasReplays: boolean = false) {
         Router.createRoutes(routes()),
         Router.match as any
       ),
+      idleTimeout: 5000,
```

Comment:
This is obviously fine.

There is one caveat though: if a user closes a page earlier than the "5s idle" (which means more than "5s on a stopwatch"), the transaction is lost.

Reply:
@rhcarvalho I wonder if we can send envelopes on page unload?

Downside is that this will be no longer permissible by default on Chrome: https://www.chromestatus.com/feature/4664843055398912 

`navigator.sendBeacon()` will be preferred.

Reply:
There's a limit of (IIRC) 64 KB for beacon / fetch keep-alive, plus differences in browser support to be taken into account. Generally I want us to move towards those APIs when possible.